classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java

```diff
@@ -34,9 +34,9 @@
     @Autowired
     private ClassificatorService classificatorService;
     @Autowired
-    private ClassificatorItemService itemService;
+    private SearchService searchService;
```

Comment:
How is this change related?

Reply:
Unrelated, but it's ok to have small simple clean-ups unrelated to PR's, because there's a lot to clean)